backend/app/services/attachment/attachment_service.py

```
@@ -215,35 +239,88 @@ def delete_attachment(
             logger.warning(
                 f"Cannot delete attachment {attachment_id}: linked to subtask {attachment.subtask_id}"
             )
             return False
-        
+
+        # Delete from MinIO if stored there
+        if attachment.minio_object_key:
+            try:
+                minio_client.delete_file(attachment.minio_object_key)
+                logger.info(f"Deleted attachment {attachment_id} from MinIO: {attachment.minio_object_key}")
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)